### ISSUE TITLE
Stop RBM and reboot PC schedule

### ISSUE THREAD

**Issue by astronyu** [RainbowMiner/RainbowMiner#1124]
@RainbowMiner Is there a safe and automated way to stop RBM and reboot PC on certain schedule?
Sometimes my rig stopped communicating with MRR and it shows as offline over there, the only way to get it back online usually by rebooting my rig.

**Comment by RainbowMiner** [comment]
Yes, that should be possible. There is already a reboot feature build-in, but not yet connected to any shedule.
